xds/internal/balancer/edsbalancer/eds_impl_priority.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
+var logger = grpclog.Component("xds")
```

Comment:
The xds logging is done using this https://github.com/grpc/grpc-go/blob/master/internal/grpclog/prefixLogger.go

The xDS logging prints not only the component name, also xDS type (CDS or EDS, or other), and the address. We need to keep that.

We can make a wrapper on top of component, that stores the prefix.

This is true for all xDS components.

Reply:
You can leave all xds packages as is in this PR, and update them separately.

Reply:
Ping: will/should these files be reverted?

Reply:
Keeping (or removing) is fine. I will make a follow up PR to fix all xds logging to use component. They are missing the `[xds]` prefix now.